### ISSUE TITLE
Some missing countries

### ISSUE THREAD

**Issue by Unlearn** [datasets/geo-boundaries-world-110m#1]
Not an exhaustive list of missing countries, but might be helpful for others to see what data is missing: 

```
Andorra
Antigua and Barbuda
Bahrain
Barbados
Cook Islands
Hawaii
Hong Kong
Liechtenstein
Maldives
Malta
Marshall Islands
Mauritius
Monaco
Nauru
Norfolk Island
Palau
Republic of Moldova
Saint Kitts and Nevis
Saint Lucia
Saint Vincent and the Grenadines
Samoa
Seychelles
Singapore
Tahiti
Tonga
Tuvalu
Zanzibar
```

**Comment by rufuspollock** [comment]
@Unlearn thanks for flagging this. Is this because they are too small to show up at 1:110m or is there a bug in our pulling of data from natural earth?

**Comment by Unlearn** [comment]
Hi @rgrp forgive my ignorance of how this GeoJSON was gathered from the Natural Earth data. I am unsure how to verify the visibility of a country or land-mass at this specific ratio. It's safe to assume this isn't a bug.

The README states "Geodata data package providing geojson polygons for all the world's countries", perhaps this could be updated to specify that some countries are missing from the data set because they're not visible a this ratio? This may avoid confusion for others in the future.

**Comment by jalbertbowden** [comment]
looking at the geojson, they're clearly missing.
looking at natural earth data, the admin 0 countries level, it has 247 countires, while the admin 0 details level has 197 sovereign states. i'm guessing the details level was used. looking more into it atm.

**Comment by rufuspollock** [comment]
@lexman would you like to check this item?

**Comment by lexman** [comment]
The countries are definitly missing in the source data. I opened an issue in the dataset : https://github.com/datasets/geo-countries/issues/1

**Comment by ghost** [comment]
> Not an exhaustive list of missing countries, but might be helpful for others to see what data is missing:
> 
> ```
> Andorra
> Antigua and Barbuda
> Bahrain
> Barbados
> Cook Islands
> Hawaii
> Hong Kong
> Liechtenstein
> Maldives
> Malta
> Marshall Islands
> Mauritius
> Monaco
> Nauru
> Norfolk Island
> Palau
> Republic of Moldova
> Saint Kitts and Nevis
> Saint Lucia
> Saint Vincent and the Grenadines
> Samoa
> Seychelles
> Singapore
> Tahiti
> Tonga
> Tuvalu
> Zanzibar
> ```

Hi , 
I am new to the github website.
Apart from the aforementioned list, Bermuda is also missing as per the following link:
https://raw.githubusercontent.com/datasets/geo-boundaries-world-110m/master/countries.geojson 

Secondly, I am working on my first project on an interactive map through bokeh (https://bokeh.pydata.org/en/latest/)
I have no clue to write/update a Json or a geoJson file.
So, I am adding the list of co-ordinates of the missing countries manually in my script

@rufuspollock, 
since I would modify your map in my script, shall I cite the following link on the map? -- https://github.com/datasets/geo-boundaries-world-110m/blob/master/countries.geojson

Looking forward for your suggestions


Regards
Rachit

**Comment by jalbertbowden** [comment]
would you mind sharing your work here after you publish? interested to see what you are doing/using.

**Comment by ghost** [comment]
I will try to publish a sample of my work after 6 months, subject to my employer's permission.

**Comment by rufuspollock** [comment]
@jalbertbowden could you get these corrections in? or is that hard?

**Comment by jalbertbowden** [comment]
Unless I'm mistaken, this repo is deprecated; the README.md points to https://github.com/datasets/geo-countries.  
Should I still try to add them? Feels like I shouldn't, but I'm not sure.  
Did you consider using the other dataset @rachitchauhan2?  
Happy to help, just seeking clarificaiton.

**Comment by rufuspollock** [comment]
@jalbertbowden yes can we get corrections into geo-countries. There is also a PR there if you could look at it.

**Comment by jalbertbowden** [comment]
@rufuspollock it looks like they're using the 10m cultural vectors dataset, which has most of these countries that are listed as missing.  
@rachitchauhan2 Do you mean the US state of Hawaii? Is Moldova the same as the Republic of Moldova?  
10m dataset compared with missing countries list only has these missing:  
-Hawaii
-Republic of Moldova
-Tahiti
-Zanzibar
